src/site/content/en/blog/multi-origin-pwas/index.md

```diff
@@ -96,20 +94,13 @@ Since there's no way to share permissions across origins, the only solution here
-
 There's also the issue that the same user could receive multiple installation prompts when navigating the site, if each subdomain meets the [installation criteria](https://developers.google.com/web/fundamentals/app-install-banners/#criteria), and prompts the user to install the PWA.
 
-Mitigate this problem by applying any of the following techniques:
-
-- **Showing the prompt on a single domain:** If the site has many subdomains that pass the installation criteria, the `beforeinstallprompt` event could be used and `preventDefault()` called (as explained [here](https://developers.google.com/web/fundamentals/app-install-banners/#listen_for_beforeinstallprompt)), to prevent the prompt from appearing in unintended parts of the site (like subdomains), while continue to show it in other parts (e.g. the home page).
-
-- **Showing the prompt once, across different subdomains:** It's also possible to let each subdomain show the installation prompt. (This can make sense if some subdomains are more frequently visited). In this case, it's important to avoid showing the prompt to a user multiple times. To that purpose, a cookie could be used, to track if the prompt has already been shown, and check for its existence when the `beforeinstallprompt` event is triggered. If the cookie is present, it means that the user already received the prompt somewhere else, so `preventDefault()` can be called, to avoid showing it again.
-
+To mitigate this problem you can make sure that the prompt is shown only on a single domain. If the site has many subdomains that pass the installation criteria, the `beforeinstallprompt` event could be used and `preventDefault()` called (as explained [here](https://developers.google.com/web/fundamentals/app-install-banners/#listen_for_beforeinstallprompt)), to prevent the prompt from appearing in unintended parts of the site (like subdomains), while continue to show it in other parts (e.g. the home page).
 
 ### Standalone Mode
 
-While navigating in a standalone window, the browser will behave differently when the user moves outside of the scope set by the PWA's manifest. The behavior depends on each browser version and vendor. For example, the latest Chrome versions open a [Chrome custom tab](https://developer.chrome.com/multidevice/android/customtabs), when a user moves out of the scope in fullscreen mode.
+While navigating in a standalone window, the browser will behave differently when the user moves outside of the scope set by the PWA's manifest. The behavior depends on each browser version and vendor. For example, the latest Chrome versions open a [Chrome custom tab](https://developer.chrome.com/multidevice/android/customtabs), when a user moves out of the scope in standalone mode.
```

Comment:
```suggestion
While navigating in a standalone window, the browser will behave differently when the user moves outside of the scope set by the PWA's manifest. The behavior depends on each browser version and vendor. For example, the latest Chrome versions open a [Chrome Custom Tab](https://developer.chrome.com/multidevice/android/customtabs), when a user moves out of the scope in standalone mode.
```

`Chrome Custom Tab` should be capitalized

FYI the link text you provided `Chrome Custom Tab` is an example of good link text because it matches the title of the document